src/plots/mapbox/layout_attributes.js

```diff
@@ -48,10 +48,16 @@ var attrs = module.exports = overrideAll({
         dflt: constants.styleValueDflt,
         role: 'style',
         description: [
-            'Sets the Mapbox map style.',
-            'Either input one of the default Mapbox style names or the URL to a custom style',
-            'or a valid Mapbox style JSON.',
-            'From OpenStreetMap raster tiles, use *open-street-map*.'
+            'Sets the Mapbox base map style.',
+            'Base map styles are rendered below all traces and layout layers.',
```

Comment:
I would include a bit more detail here:
* Base map styles can include multiple layers
* Base map layers are always rendered below traces and layers defined in the adjacent `layout.mapbox.layers` attribute

Reply:
For my own edification: is it possible to use the new `below` attribute to render `choroplethmapbox` traces below layers specified in this `layout.mapbox.style.layers` object or do we somehow constrain the behaviour of `below` to only look at those in `layout.mapbox.layers` ?

Reply:
Tangent on a tangent: with the new `*mapbox` trace types, how does inter-trace rendering order work? Should `below` be added to all three `mapbox` traces?

Reply:
> is it possible to use the new `below` attribute to render `choroplethmapbox` traces below layers specified in this `layout.mapbox.style.layers`

Yes.

> how does inter-trace rendering order work

like this:

https://github.com/plotly/plotly.js/blob/14133af4fc205a7d00b158bb91e4afec845fd1d3/src/plots/mapbox/mapbox.js#L200-L204

`densitymapbox` has a `below` attribute:

https://github.com/plotly/plotly.js/blob/14133af4fc205a7d00b158bb91e4afec845fd1d3/src/traces/densitymapbox/attributes.js#L67-L79

only `scattermapbox` does not. I didn't think that would be necessary. I can add `below` to `scattermapbox` if you want.

Reply:
I think for completeness and symmetry would should add `below` to `scattermapbox`, yes.

Reply:
Now in https://github.com/plotly/plotly.js/issues/4049